psutil/_psbsd.py

```diff
@@ -60,6 +61,15 @@
         cext.SRUN: _common.STATUS_WAKING,
         cext.SONPROC: _common.STATUS_RUNNING,
     }
+elif NETBSD:
+    PROC_STATUSES = {
+        cext.SIDL: _common.STATUS_IDLE,
+        cext.SACTIVE: _common.STATUS_RUNNING,
+        cext.SDYING: _common.STATUS_ZOMBIE,
```

Comment:
Mmmm I'm not so sure about this. Also can't find any documentation. Have you ever bumped into this status?